src/views/EditSidebar.vue

```diff
@@ -109,6 +109,7 @@
 				:is-read-only="isReadOnly"
 				:prop-model="rfcProps.description"
 				:value="description"
+				:is-description="true"
```

Comment:
this change is needed, because when you click the textarea it jumbs to a smaller side, and i fixed it some time ago for simple editor and forgot to add it here